.github/workflows/ci.yml

```
@@ -35,6 +38,10 @@ jobs:
 
     if: github.repository_owner == 'joelbenway'
 
+    ignore:
```

actionlint (1.7.4)

unexpected key "ignore" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)